docs/source/internal/feature_extraction_utils.rst

```diff
@@ -0,0 +1,29 @@
+.. 
+    Copyright 2021 The HuggingFace Team. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+    the License. You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+    an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+    specific language governing permissions and limitations under the License.
+
+Utilities for Feature Extractors
+-----------------------------------------------------------------------------------------------------------------------
+
+This page lists all the utility functions used by the feature extractors, mainly the class
+:class:`~transformers.feature_extraction_utils.PreTrainedFeatureExtractor`.
+
+Most of those are only useful if you are studying the code of the feature extractors in the library.
+
+
+Enums and namedtuples
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. autoclass:: transformers.file_utils.ExplicitEnum
```

Comment:
I'm surprised sphinx lets you document the same object twice as I don't know to which place it will point you when using the :class:\`~transformers.file_utils.ExplicitEnum\` pointers.

I thin we should create a new internal doc page to document everything in file_utils (we can add our special decorators after for instance).